tests/integration/assessment/test_workflows.py

```diff
@@ -21,3 +21,44 @@ def test_running_real_assessment_job(ws, installation_ctx, make_cluster_policy,
 
     after = installation_ctx.generic_permissions_support.load_as_dict("cluster-policies", cluster_policy.policy_id)
     assert after[ws_group_a.display_name] == PermissionLevel.CAN_USE
+
+
+@retried(on=[NotFound, InvalidParameterValue], timeout=timedelta(minutes=8))
+def test_running_assessment_job_crawl_tables(
+    ws, installation_ctx, make_cluster_policy, make_cluster_policy_permissions, make_schema, make_table
+):
+    ws_group_a, _ = installation_ctx.make_ucx_group()
+    cluster_policy = make_cluster_policy()
+    make_cluster_policy_permissions(
+        object_id=cluster_policy.policy_id,
+        permission_level=PermissionLevel.CAN_USE,
+        group_name=ws_group_a.display_name,
+    )
+    source_schema = make_schema(catalog_name="hive_metastore")
+    managed_table = make_table(schema_name=source_schema.name)
+    external_table = make_table(schema_name=source_schema.name, external=True)
+    tmp_table = make_table(schema_name=source_schema.name, ctas="SELECT 2+2 AS four")
+    view = make_table(schema_name=source_schema.name, ctas="SELECT 2+2 AS four", view=True)
+    non_delta = make_table(schema_name=source_schema.name, non_delta=True)
+
+    installation_ctx.__dict__['include_object_permissions'] = [f"cluster-policies:{cluster_policy.policy_id}"]
+    installation_ctx.workspace_installation.run()
+
+    installation_ctx.deployed_workflows.run_workflow("assessment")
+
+    tables_crawled = []
+    table_list = list(
+        installation_ctx.sql_backend.fetch(
+            f"SELECT * FROM {installation_ctx.inventory_database}.tables WHERE database='{source_schema.name}'"
+        )
+    )
+    for table in table_list:
+        tables_crawled.append(table.name)
```

Comment:
```suggestion
    tables_crawled = installation_ctx.tables_crawler.snapshot()
```

source database should be propagated down via `include_databases` property. we already have one test that runs assessment workflow - extend that instead